workspaces/analytics/.changeset/shiny-scissors-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-analytics-provider-segment': patch
```

Comment:
I recommend starting with a patch=0 release after migrating a plugin. Also if both plugins have different package names and will never conflict, it will also allow us to have patch releases on the old package.

```suggestion
'@backstage-community/plugin-analytics-provider-segment': minor
```

Reply:
Hmm, we have been doing patch updates for migrations so far. Do you think we should continue sticking to that?